inscriptions/ISic0941.xml

```diff
@@ -170,33 +175,32 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI140422">
-                <ab>
-                    <lb n="1"/>
-
-
-                     α 𐆠 ω<lb n="2"/>Οὐλπία δέξε <surplus>²⁶δέξαι</surplus>²⁶<lb n="3"/>τεκοῦσα Θεοκτίστην<lb n="4"/>
-                     λαγόνεσσιν.<lb n="5"/>
-                    <lb n="6"/>εὐμοίρει
-                    <lb n="7"/>
-                    <supplied reason="lost">Οὐλπία</supplied>.<lb n="8"/>εὐμοίρει<lb n="9"/>Θεοκτίστη.
-                    <lb n="10"/>
-                </ab>
-            </div>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI333255">
-                <ab>
-                    <lb n="1"/>
-
-
-                    εὐμοίρει.<lb n="2"/>
-                    <lb n="3"/>εὐμοίρει Θεοκτίστη.<lb n="4"/>
-                    <lb n="5"/> Α 𐆠 Ω<lb n="6"/>Οὐλπία δέξε<lb n="7"/>τεκοῦσα Θεοκτίστην<lb n="8"/>
-                    λαγόνεσ<supplied reason="lost">σι</supplied>ν.
-                    <lb n="9"/>
-                </ab>
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#VM">
```

Comment:
I haven't seen Ferrua 1940 (but have just requested a scan from the Bodleian), but after looking at Agnello, I would simply use milestone rather than textpart, and have n="a", "b", and "c", in teh same sequence as Kaibel, since this seems to be a text in the top of an arch (b) with (a) and (c) to its left and right, and to belong all together by its apparent content, and best here just presented in its left-right sequence, as does Kaibel.